README.md

```diff
@@ -42,39 +42,14 @@ Eventually, the plan is to have self-contained packages for \*nix systems and an
 ### Step 1 - Login (aka add a project)
 
 ```
-twilio project:add
+twilio login
 ```
 
-which is an alias for:
+This is for caching your credentials for your _existing_ Twilio account (aka Project) locally. You will be prompted for a shorthand identifier for your Twilio project. This is referred to as the "project ID" which is just an easy to remember, short string to identify the project. (If you've used `git` before, it's like the name you assign to a remote like "origin".)
```

Comment:
"This is for caching your credentials for your _existing_ Twilio account (aka Project) locally." -> "This command caches your _existing_ Twilio account (aka Project) credentials."

"This is referred to as the "project ID" which is just an easy to remember, short string to identify the project." -> "This is referred to as the "project ID", which is just an easy to remember, short string to identify the project."

Reply:
Is the second line just adding the comma?

Reply:
For the first line, it's not caching existing credentials. It uses existing credentials to cache new ones.

Reply:
👍 on the first line.

Yes, Just adding a comma.